### PR TITLE
Fixed exceptional bounce between physics bodies which should be standstill on the floor

### PR DESCRIPTION
**Pull request by WenhaiLin**

### CODE REVIEW

File: cocos/physics/CCPhysicsWorld.h [hunk 1]
```diff
@@ -131,26 +131,33 @@ class CC_DLL PhysicsWorld
...
+    float getTimeStep() { return _timeStep; }
+    
     /**
      * set the update rate of physics world, update rate is the value of EngineUpdateTimes/PhysicsWorldUpdateTimes.
      * set it higher can improve performance, set it lower can improve accuracy of physics world simulation.
      * default value is 1.0
      * Note: if you setAutoStep(false), this won't work.
      */
-    inline void setUpdateRate(int rate) { if(rate > 0) { _updateRate = rate; } }
+    CC_DEPRECATED_ATTRIBUTE void setUpdateRate(int rate);
```
Comment (minggo) [inline cocos/physics/CCPhysicsWorld.h]: Please use CC_DEPRECATED(v4)

File: cocos/physics/CCPhysicsWorld.h [hunk 2]
```diff
@@ -131,26 +131,33 @@ class CC_DLL PhysicsWorld
...
-    inline void setSpeed(float speed) { if(speed >= 0.0f) { _speed = speed; } }
+    inline void setSpeed(float speed) { if(speed > 0.0f) { _speed = speed; } }
     /** get the speed of physics world */
     inline float getSpeed() { return _speed; }
+
+    /** Set the time step of physics world. default value is 0.02
+     * Note: if you disable auto step, this won't work.
+     */
+    void setTimeStep(float timeStep);
+    float getTimeStep() { return _timeStep; }
```
Comment (minggo) [inline cocos/physics/CCPhysicsWorld.h]: Please add const

**Comment by WenhaiLin** [comment]
[ci rebuild]

**Comment by minggo** [comment]
Please modify corresponding lua test.

**Pull request merged**